epidose/device/supervisord.conf

```diff
@@ -26,10 +26,10 @@ redirect_stdout=true           ; redirect proc stderr to stdout (default false)
 stderr_logfile=/var/log/%(program_name)s ; stderr log path, NONE for none; default AUTO
 stderr_logfile_maxbytes=1MB    ; max # logfile bytes b4 rotation (default 50MB)
 
-[program:watchdog]
-command=/opt/venvs/epidose/bin/python /opt/venvs/epidose/lib/python3.7/site-packages/epidose/device/watchdog_d.py
+[program:update_filter]
```

Comment:
Why are you moving programs around?  Please avoid unneeded changes, because they make reviewing move difficult.  If some refactoring is needed, please make a separate PR.

Reply:
I thought of sorting them alphabetically, that is why I moved them around.
Ok, I will issue a PR about it later.